07-routing-gossip.md

```diff
@@ -109,6 +109,11 @@ Thus the hash skips the 4 signatures, but hashes the rest of the message, includ
 The creating node SHOULD set `len` to the minimum length required to
 hold the `features` bits it sets.
 
+The receiving node MUST verify the integrity and authenticity of the message by verifying the signatures.
+The receiving node MUST check that the `features` field does not contain any unknown even bits.
```

Comment:
I try to avoid saying "MUST check", because it doesn't say what to do if the check fails.  In fact, this sentence can be eliminated.